packages/discovery-provider/src/queries/comments/utils.py

```diff
@@ -444,3 +459,140 @@ def build_comments_query(
         )
 
     return query
+
+
+def format_comments(
+    session, comments, current_user_id, include_replies=False, artist_id=None
+):
+    """
+    Format a list of comments, optionally including their replies.
+
+    Args:
+        session: Database session
+        comments: List of comment query results (tuples from the database)
+        current_user_id: ID of the user making the request
+        include_replies: Whether to include replies in the response
+        artist_id: ID of the track owner (for track comments and replies)
+
+    Returns:
+        List of formatted comments
+    """
+    from src.queries.comments.replies import get_replies
```

Comment:
I'm so curious why we defer these imports to inside a function instead of just importing at the top of the file...

Reply:
AI was doing that I was also surprised

Reply:
I'll move them up though